src/main/java/com/walking/intensive/chapter3/task14/Task14.java

```diff
@@ -43,11 +45,49 @@
  */
 public class Task14 {
     public static void main(String[] args) {
-//        Для собственных проверок можете делать любые изменения в этом методе
+//        int[][] obj = {
+//                {1, 3},
+//                {3, 3},
+//                {5, 3},
+//                {2, 2}};
+//        int[][] radar = {
+//                {1, 3, 1},
+//                {4, 3, 1},
+//                {1, 1, 2}};
+//        System.out.println("== Объекты ==");
+//        printArray(obj);
+//        System.out.println("== Радары ==");
+//        printArray(radar);
+//        System.out.println("== Объекты на Радарах ==");
+//        getObjectCounts(obj, radar);
     }
 
+//    static void printArray(int[][] array) {
+//        for (int[] rows : array) {
+//            for (int elements : rows) {
+//                System.out.print(elements + " ");
+//            }
+//            System.out.println();
+//        }
+//    }
+
     static int[] getObjectCounts(int[][] objectLocations, int[][] radars) {
-        // Ваш код
-        return new int[0];
+        int[] objectCounts = new int[radars.length];
+        for (int j = 0; j < radars.length; j++) {
+            if (radars[j].length != 3 || radars[j][2] <= 0) {
+                return new int[0];
+            }
+            int counter = 0;
+            for (int[] objectLocation : objectLocations) {
+                if (objectLocation.length != 2) {
+                    return new int[0];
+                }
+                if ((Math.pow(objectLocation[0] - radars[j][0], 2) + Math.pow(objectLocation[1] - radars[j][1], 2)) <= Math.pow(radars[j][2], 2)) {
```

Comment:
> Task :checkstyleMain
Error: eckstyle] [ERROR] /home/runner/work/intensive-tasks-2024/intensive-tasks-2024/src/main/java/com/walking/intensive/chapter3/task14/Task14.java:85: Line is longer than 120 characters (found 147). [LineLength]